crnk-integration-examples/openliberty-microprofile-example/build.gradle

```diff
@@ -21,7 +21,7 @@ dependencies {
 	providedCompile 'org.eclipse.microprofile:microprofile:4.0.1'
 
 	implementation project(':crnk-setup:crnk-setup-rs')
-	implementation project(':crnk-setup:crnk-setup-cdi')
+	// implementation project(':crnk-setup:crnk-setup-cdi')
```

Comment:
This is temporary - I was having issues with CDI and decided to focus on other parts. Welcome to any suggestions.